cheroot/test/helper.py

```diff
@@ -40,7 +41,7 @@ class CherootWebCase(webtest.WebCase):
     scheme = 'http'
 
     available_servers = {
-        'wsgi': cheroot.server.WSGIServer,
+        'wsgi': cheroot.wsgi.WSGIServer,
```

Comment:
Yes, but it seems those names are added for backwards compatibility and may be a subject of deprecation in future. So it's always better to use real names.

@jaraco am I missing anything?